app/Http/Requests/RegisterUserRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Http\Requests;
+
+class RegisterUserRequest extends BaseRequest
+{
+    public function rules(): array
+    {
+        return [
+            "email" => [
+                "required",
+                "email",
+                "unique:users",
+            ],
+            "password" => [
```

Comment:
`password` section is almost identical in `LoginUserRequest`. Changing these 8 minimum characters would require editing at least two files... if developer would even know that.